src/Joblify.Core.Data/UnitOfWork/UnitOfWork.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Joblify.Core.Data.Context;
+using Joblify.Core.Data.Models;
+using Joblify.Core.Data.Repository;
+using Joblify.Core.Data.Repository.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace Joblify.Core.Data.UnitOfWork
+{
+    public class UnitOfWork : IUnitOfWork
+    {
+        private readonly JoblifyDbContext _dbContext;
+        public IRepository<User> UserRepository => new GenericRepository<User>(_dbContext);
+        public IRepository<File> FileRepository => new GenericRepository<File>(_dbContext);
+        public IRepository<Role> RoleRepository => new GenericRepository<Role>(_dbContext);
+        public IRepository<ExternalProvider> ExternalProviderRepository => new GenericRepository<ExternalProvider>(_dbContext);
+
+        public UnitOfWork(JoblifyDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public void Commit()
+        {
+            _dbContext.SaveChanges();
+        }
+
+        public void Dispose()
+        {
+            _dbContext.Dispose();
+        }
+
+        public void RejectChanges()
```

Comment:
What is use case of this method?

Reply:
tbh i paste example of method from [LINK](https://medium.com/@utterbbq/c-unitofwork-and-repository-pattern-305cd8ecfa7a). I thought that it might be useful in the future.

Reply:
Oh I see, rather than implementing most of things (refering to my question about update), we rely more on EF Core, that's ok.